nautobot/core/tables.py

```diff
@@ -557,7 +557,7 @@ def render(self, *, bound_column, record, value):  # pylint: disable=arguments-d
                 related_record = related_records[0]
         url = reverse(self.viewname, kwargs=self.view_kwargs)
         if self.url_params:
-            url += "?" + urlencode({k: getattr(record, v) for k, v in self.url_params.items()})
+            url += "?" + urlencode({k: (getattr(record, v) or "") for k, v in self.url_params.items()})
```

Comment:
```suggestion
            url += "?" + urlencode({k: (getattr(record, v, "")) for k, v in self.url_params.items()})
```

Reply:
This would not work in this case as `record` has the attribute `v` but the attributes value is `None`, this would only work when `record` do not have the attribute `v`